seqr/views/apis/dashboard_api_tests.py

```diff
@@ -120,3 +122,116 @@ def test_export_projects_table(self):
         response = self.client.get('{}?file_format=csv'.format(url))
         self.assertEqual(response.status_code, 400)
         self.assertDictEqual(response.json(), {'error': 'Invalid file_format: csv'})
+
+
+# Tests for AnVIL access disabled
+class LocalDashboardPageTest(AuthenticationTestCase, DashboardPageTest):
+    fixtures = ['users', '1kg_project']
+
+
+# Test for permissions from AnVIL only
+@mock.patch('seqr.views.utils.terra_api_utils.TERRA_API_ROOT_URL', TEST_TERRA_API_ROOT_URL)
+class AnvilDashboardPageTest(AnvilAuthenticationTestCase, DashboardPageTest):
+    fixtures = ['users', 'social_auth_data', '1kg_project']
```

Comment:
We don't usually include "data" in the fixture name, since fixtures are definitionally data, so I would change the filename from`social_auth_data` to `social_auth`

Reply:
Okay